.github/workflows/update-test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Update test
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
+  push:
+    branches:
+      - prerelease_test
+  pull_request:
+jobs:
+  update_test:
+    name: Update test PG${{ matrix.pg }}
+    runs-on: 'ubuntu-18.04'
+    strategy:
+      matrix:
+        pg: ["11.0","12.0"]
+        include:
+          - pg: "11.0"
+            pg_major: 11
+          - pg: "12.0"
+            pg_major: 12
+      fail-fast: false
+    env:
+      PG_VERSION: ${{ matrix.pg }}
+    steps:
+    - name: Checkout TimescaleDB
+      uses: actions/checkout@v2
+
+    - name: Update tests ${{ matrix.pg }}
+      run: sudo ./scripts/test_updates_pg${{ matrix.pg_major}}.sh
```

Comment:
Why do we need to run these as `root` now? I can't see that Travis did this and I have never had to do it when running locally.

I think, generally, that we should avoid running tests with elevated privileges because it might mask issues that might otherwise occur.